src/EficazFramework.sln

```diff
@@ -19,6 +19,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SolutionScripts", "Solution
 		SolutionScripts\Tests.psm1 = SolutionScripts\Tests.psm1
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ConsoleTests", "ConsoleTests\ConsoleTests.csproj", "{4619A0CD-6E64-495D-8449-B386E6C57451}"
```

Comment:
Não compreendi qual seria a idéia acerca da criação deste projeto ConsoleTests, mas se for exclusivamente executar testes, a Solution já possui um projeto para execução de testes.

https://github.com/Eficaz-Sistemas/EficazFramework.SPED/tree/master/src/Tests/EficazFramework.Tests

Reply:
Opa, tudo bem? Isso foi excluído. Foi utilizado para testar manualmente um método para entender como funciona e forçar um erro. Acho que ficou um lixo no código.